src/errors/error-messages.ts

```diff
@@ -24,6 +24,7 @@ export const erc20CollateralPoolErrorMessage = {
   maxLimitAllowed: 'Max limit allowed is 1000',
   endTimeReached: 'Pool has ended',
   poolIsNotClosed: 'Pool is not closed',
+  maxLendedIsReached: 'Max lended is reached',
```

Comment:
Past tense of Lend is Lent 

Max Lent

Reply:
maxLendedIsReached: 'Maximum lending limit reached'

Reply:
Sure, you're right, I followed same contract names but I'll correct it, thanks.